src/pages/productPage/componets/filter/filter.js

```diff
@@ -27,7 +27,7 @@ export const Filter = () => {
               dispatch({ type: "sortBy", payload: "LOW_TO_HIGH" })
```

Comment:
this type can be put inside a separate file.
----------
actions.js
`const actions = { SORTBY:"sortBy"}`
`export default actions;`
---------
and it can be imported in to both context & filter.js

Reply:
I'll look into that